### ISSUE TITLE
'Text color' and 'Background color' popup are not accessible through keyboard present under 'Body' field.

### ISSUE THREAD

**Issue by Ram-G** [Azure/LogicAppsUX#4457]
### Describe the issue 

1. Open designer to an action that uses HTML editor. 
2. Observe that user is unable to tab onto the color palette popup. 

### Are you using new designer or old designer 

New Designer (With action panels)

### Screenshots or Videos 

![image](<https://github.com/Azure/LogicAppsUX/assets/10837129/270c52bd-ed9d-435e-9983-55574ffa946e>)

### Browser 

Edge

### Additional context 

ADO link: [https://msazure.visualstudio.com/One/\_workitems/edit/27261167](<https://msazure.visualstudio.com/One/_workitems/edit/27261167>)
